### PR TITLE
updated styles

### PR DESCRIPTION
**Pull request by 311chaos**
- stole styles from http://sliptree.github.io/bootstrap-tokenfield/

@zack37 - Can you take a look and pull this into master if this is ok?

### CODE REVIEW

**Comment by 311chaos** [comment]
@zack37 - Could you take a look?

File: ll-token-field.html [hunk 1]
```diff
@@ -110,18 +167,32 @@
         e.stopPropagation();
         var newTag = input.value;
         input.value = '';
-        this.tags = this.tags.concat(newTag);
+        this.push('tags', newTag);
+        this.tagChanged();
         input.focus();
       }
       else if (e.keyCode === 8 && input.value === '') { // Backspace
```
Comment (zack37) [inline ll-token-field.html:174]: I know this condition was already here, and it's my fault, but we should probably make sure the string isn't whitespace as well as empty so we don't end up with empty tags.

File: demo/index.html [hunk 2]
```diff
@@ -16,11 +7,18 @@
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
     <link rel="import" href="../ll-token-field.html">
   </head>
-  <body>
+  <body style="margin: 30px;">
 
     <p>An example of <code>&lt;ll-token-field&gt;</code>:</p>
 
-    <ll-token-field tags='["1", "2", "3", "4"]'></ll-token-field>
+    <div style="height: 100px; border:1px solid gainsboro; border-radius: 4px; padding: 5px;">
```
Comment (zack37) [inline demo/index.html:14]: What was the intention of putting the border around it?

Comment (311chaos) [inline demo/index.html:14]: Yes, I wanted the hosting item to control the border. For display purposes, I added  a wrapper with these styles to the demo.

I might be wrong with this implementation, but it did work for our first use case. If you have any ideas on this, I am open to them.

File: ll-token-field.html [hunk 3]
```diff
@@ -110,18 +167,32 @@
...
     },
+
+    _focusInput: function() {
+      var input = this.$$('#tag-input');
+      input.focus();
+    },
+    tagChanged: function() {
+      //needing to hard reset the tags object to get it reflected to the attribute
+      this.tags = this.tags.splice(0);
+      this.fire('ll-token-modified', this.tags)
```
Comment (zack37) [inline ll-token-field.html:194]: I think we should add a test for this method to make sure the event being fired passes the correct value up to its parent

File: ll-token-field.html (hunk 1 above)
Comment (311chaos) [inline ll-token-field.html:174]: Added a regex check that should fail the input if it is just whitespace. I also added a test to validate the failure.

File: ll-token-field.html (hunk 3 above)
Comment (311chaos) [inline ll-token-field.html:194]: I added a test for the removal of the tokens, it ensures that the event is firing with the proper data passed. I also have tests for this on the ll-property-image (for both adding and removing tags to ensure I get the events)

**Comment by 311chaos** [comment]
Changes made. One last peek @zack37 ?

File: ll-token-field.html (hunk 3 above)
Comment (311chaos) [inline ll-token-field.html]: Seems to be fine. I think this came from when I was working through an issue with a watch not firing, however the root cause of that issue ended up being something different.

**Comment by zack37** [comment]
:+1:  Going to leave this open for a bit so others can chime in if they want to

**Pull request merged**